test/cli/test-files.js

```diff
@@ -72,43 +74,116 @@ describe('files', () => {
       })
     })
 
-    it('add recursively test 1', () => {
-      return ipfs('files add -r src/init-files/init-docs').then((out) => {
+    it('add recursively test', () => {
+      return ipfs('files add -r test/test-data/recursive-get-dir').then((out) => {
```

Comment:
Thank you for creating a folder that will stay immutable, this will save time in the future :)